community/cypher/cypher-logical-plans-3.4/src/main/scala/org/neo4j/cypher/internal/v3_4/logical/plans/Expand.scala

```diff
@@ -28,15 +28,7 @@ import org.neo4j.cypher.internal.v3_4.expressions._
   * provided constraints. Produce one row per traversed relationships, and add the
   * relationship and end node as values on the produced rows.
   */
-case class Expand(source: LogicalPlan,
-                  from: String,
-                  dir: SemanticDirection,
-                  types: Seq[RelTypeName],
-                  to: String,
-                  relName: String,
-                  mode: ExpansionMode = ExpandAll)
-                 (val solved: PlannerQuery with CardinalityEstimation)
-                 (implicit idGen: IdGen)
+case class Expand(source: LogicalPlan, from: String, dir: SemanticDirection, types: Seq[RelTypeName], to: String, relName: String, mode: ExpansionMode = ExpandAll)(implicit idGen: IdGen)
```

Comment:
I would suggest not putting all the arguments on one line here, because it becomes much harder to read. Auto-format mishap? Also in more places in this file.